Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
```

Comment:
commit message: you mean "comdat-folded"?

Reply:
Thank you, fixed